### DISCUSSION TITLE
Use preload in webpack_encore

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#48131] [Q&A]
Hi. I use `webpack_encore` and `preload` in my project.
I add `preload: true` to my `config/packages/webpack-encore.yaml` file.
When I run my site in local, show this warning in my console:

> The resource <URL> was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.

How can I fix this problem?

**Comment by MatTheCat** [comment]
Hi,

this should not happen; could you provide a [reproducer](https://symfony.com/doc/current/contributing/code/reproducer.html)?
